Stripe/UI/STPPaymentCardTextField.m

```diff
@@ -403,6 +403,55 @@ - (STPCardParams *)card {
     return c;
 }
 
+- (void)setCard:(nonnull STPCardParams *)card {
```

Comment:
Note that these tests don't cover the cases when the field is first responder – we'd need to add an application test target. (Happy to do that if you think we should).